internal/logging/rotate.go

```diff
@@ -28,7 +28,8 @@ func rotateLogs(files []fs.FileInfo, timeCutoff time.Time, amountCutoff int) []f
 	for prefix := range prefixes {
 		c := 0
 		for _, file := range files {
-			if strings.HasPrefix(file.Name(), prefix) && strings.HasSuffix(file.Name(), FileNameSuffix) {
+			currentPrefix := LogPrefixRx.FindString(filepath.Base(file.Name()))
```

Comment:
nit: I'm pretty sure the `filepath.Base` call is unnecessary. The `FileInfo` interface doesn't expose the full path. I ran into this in another PR using the `fs.FileInfo` 🤔